### ISSUE TITLE
Messages that contacts have sent in both group chats and private messages are grouped together

### ISSUE THREAD

**Issue by connervieira** [glaurent/MessagesHistoryBrowser#10]
When selecting a contact, MessagesHistoryBrowser shows all messages they've sent, including ones that they've sent in group chats, all in the same view. It would be helpful if there was a way to only view messages that contacts have sent in private one on one chats.

**Comment by glaurent** [comment]
Yes, right now the selection is too crude. I unfortunately lack time to work on it, but I'll keep the issue open to see what I can do.
